src/components/molecules/DashboardItem.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Card } from 'evergreen-ui';
+
+const DashboardItem: React.FC = () => {
+    console.log('title');
+    return <Card width={'100%'} height={'100%'} border="default" elevation={1}></Card>;
```

Comment:
Don't need to evaluate the strings as they are _simply strings_.
```suggestion
    return <Card width="100%" height="100%" border="default" elevation={1} />;
```

This is because statements wrapped in `{}` are evaluated as JS expressions.